e2e/tests/hmctsSuperUserAdministersCase_test.js

```diff
@@ -21,9 +30,12 @@ Scenario('HMCTS super user removes an order from a case', async (I, caseViewPage
   removeOrderEventPage.addRemoveOrderReason('Entered incorrect order');
   await I.completeEvent('Submit');
   I.seeEventSubmissionConfirmation(config.superUserActions.removeOrder);
-  const issuedDate = new Date(2020, 4,26, 14, 33);
-  order = {...blankOrder, ...{
-    document: 'Blank order (C21).pdf',
-  } };
+  const issuedDate = new Date(2020, 4, 26, 14, 33);
+  order = {
+    ...blankOrder,
+    ...{
+      document: 'Blank order (C21).pdf',
+    },
```

Comment:
i think spread is not needed on single property
 {
...blankOrder, 
document: 'Blank order (C21).pdf'
}

Reply:
Wasn't aware I touched that, code formatter must have not been happy with it.